File: Listas de Exercícios (C)/C/Lista 2 - Sala/3.c
#include <stdio.h>

int main()
{
    float N1, N2, D;
    
    printf("Diferença entre o maior e o menor número\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Número 1: "); scanf("%f", &N1);
    printf("Número 2: "); scanf("%f", &N2);
    
    printf("\n");
    
    if (N1>N2) {
        D=N1-N2;
        printf("%.2f - %.2f = %.2f", N1, N2, D);
    }
    else {
        D=N2-N1;
        printf("%.2f - %.2f = %.2f", N2, N1, D);
    }
    
    printf("\n");
}

File: Listas de Exercícios/Python/Python/Lista 2 - Sala/3.py
print("Diferença entre o maior e o menor número\n")

print("--Valores decimais devem ser digitados com (.)!--")

N1=float(input("Número 1: "))
N2=float(input("Número 2: "))

print("")

if (N1>N2):
    D=N1-N2
    print(f"{N1} - {N2} = {D:.2f}")
else:
    D=N2-N1
    print(f"{N2} - {N1} = {D:.2f}")

print("")